etc/internal_options.conf

```diff
@@ -334,7 +334,7 @@ agent.recv_timeout=60
 agent.remote_conf=1
 
 # Database - maximum number of reconnect attempts
-dbd.reconnect_attempts=10
+dbd.reconnect_attempts=3
```

Comment:
Please, restore this value to the one specified in the documentation that is `10`.

Reply:
Reverted!